File: 1. TwoSum/TwoSum.java
/*
 * @lc app=leetcode id=1 lang=java
 *
 * [1] Two Sum
 */

// @lc code=start
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] ind = new int[2];
        for (int i=0; i<nums.length; i++) {
            int current = nums[i];
            for (int j=i+1; j<nums.length; j++) {
                if (nums[j] == (target-current)) {
                    ind[0] = i;
                    ind[1] = j;
                }
            }
        }
        return ind;
    }
}
// @lc code=end

File: 1. TwoSum/two_sum.py
#
# @lc app=leetcode id=1 lang=python3
#
# [1] Two Sum
#

# @lc code=start
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ind = [0] * 2
        for i in range(len(nums)):
            temp = nums[i]
            for j in range(i+1, len(nums)):
                if nums[j] == (target - temp):
                    ind[0] = i
                    ind[1] = j
        return ind
        
# @lc code=end

